svm/src/loader.rs

```diff
@@ -3,12 +3,14 @@ use {
     solana_sdk::{account::AccountSharedData, pubkey::Pubkey},
 };
 
-/// Runtime callbacks for transaction processing.
-pub trait TransactionProcessingCallback {
-    fn account_matches_owners(&self, account: &Pubkey, owners: &[Pubkey]) -> Option<usize>;
-
+/// The "loader" required by the transaction batch processor, responsible
+/// mainly for loading accounts.
+pub trait Loader {
```

Comment:
Loader is heavily overloaded term due to accounts, program_cache, bpf_loader etc. Given the trait is very specific to accounts now, maybe we can find a name that correlates to accounts?